ingestion/src/metadata/ingestion/source/dashboard/redash.py

```diff
@@ -121,60 +128,75 @@ def yield_dashboard_lineage_details(
+                        to_entity = self.metadata.get_by_name(
+                            entity=Lineage_Dashboard,
+                            fqn=fqn.build(
+                                self.metadata,
+                                Lineage_Dashboard,
+                                service_name=self.config.serviceName,
+                                dashboard_name=str(dashboard_details.get("id")),
+                            ),
+                        )
+                        yield self._get_add_lineage_request(
+                            to_entity=to_entity, from_entity=from_entity
+                        )
+        except Exception as exc:
+            logger.debug(traceback.format_exc())
+            logger.warning(
+                f"Error to creating dashboard chart for {dashboard_details}: {exc}"
+            )
 
     def yield_dashboard_chart(
         self, dashboard_details: dict
     ) -> Optional[Iterable[CreateChartRequest]]:
         """
         Metod to fetch charts linked to dashboard
         """
-        for widgets in dashboard_details.get("widgets", []):
-            visualization = widgets.get("visualization")
-            yield CreateChartRequest(
-                name=widgets["id"],
-                displayName=visualization["query"]["name"]
-                if visualization and visualization["query"]
-                else "",
-                chartType=get_standard_chart_type(
-                    visualization["type"] if visualization else ""
-                ),
-                service=EntityReference(
-                    id=self.context.dashboard_service.id.__root__,
-                    type="dashboardService",
-                ),
-                chartUrl=f"/dashboard/{dashboard_details.get('slug', '')}",
-                description=visualization["description"] if visualization else "",
+        try:
+            for widgets in dashboard_details.get("widgets", []):
+                visualization = widgets.get("visualization")
+                yield CreateChartRequest(
+                    name=widgets["id"],
+                    displayName=visualization["query"]["name"]
+                    if visualization and visualization["query"]
+                    else "",
+                    chartType=get_standard_chart_type(
+                        visualization["type"] if visualization else ""
+                    ),
+                    service=EntityReference(
+                        id=self.context.dashboard_service.id.__root__,
+                        type="dashboardService",
+                    ),
+                    chartUrl=f"/dashboard/{dashboard_details.get('slug', '')}",
+                    description=visualization["description"] if visualization else "",
+                )
+        except Exception as exc:
+            logger.debug(traceback.format_exc())
+            logger.warning(
+                f"Error to creating dashboard chart for {dashboard_details}: {exc}"
```

Comment:
Please change the message to `Error to yield dashboard chart for {dashboard_details}: {exc}`. Also if possible specify the `chart_id` in the error message